samples/tutorials/hardwareInterfaces/adc/NoiseSensor/noiseDataComponent/noiseData.c

```diff
@@ -42,14 +39,17 @@ static void adcTimer
     }
 }
 
+//--------------------------------------------------------------------------------------------------
+/**
+ * Main program starts here
+ */
+//--------------------------------------------------------------------------------------------------
 COMPONENT_INIT
 {
-   LE_INFO("---------------------- ADC Reading started");
+    LE_INFO("---------------------- ADC Reading started");
 
-    le_clk_Time_t  timerInterval = {.sec = ADC_SAMPLE_INTERVAL_IN_SECONDS, .usec = 0};
-    le_timer_Ref_t adcTimerRef;
-    adcTimerRef = le_timer_Create("ADC Timer");
-    le_timer_SetInterval(adcTimerRef, timerInterval);
+    le_timer_Ref_t adcTimerRef = le_timer_Create("ADC Timer");
+    le_timer_SetMsInterval(adcTimerRef, ADC_SAMPLE_INTERVAL_IN_MILLI_SECONDS * 1000);
```

Comment:
I see that you do `* 1000` here.  As mentioned earlier, the `* 1000` belongs with the `#define`.